### PR TITLE
Blogpost:Adding configuration on how to debug Apache Camel with Sublime

### PR DESCRIPTION
**Pull request by Navyashree0923**

### CODE REVIEW

File: content/blog/2022/10/Emacsdap/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Camel Textual Route debugging in Emacs"
+date: 2022-10-07
```
Comment (apupier) [inline content/blog/2022/10/Emacsdap/index.md:3]:
```suggestion
date: 2022-10-17
```

**Review by apupier — CHANGES_REQUESTED** [review]
title of the PR is for Sublime but text is about emacs

**Comment by apupier** [comment]
> title of the PR is for Sublime but text is about emacs

ah ok, there are both. You need to update your main branch, then rebase and then repush so that we can see only the latest blogpost about sublime

**Comment by Navyashree0923** [comment]
> > title of the PR is for Sublime but text is about emacs
> 
> ah ok, there are both. You need to update your main branch, then rebase and then repush so that we can see only the latest blogpost about sublime

Okay, thanks

**Review by apupier — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

**Pull request merged**